webhook-service/lib/webhook_config.go

```diff
@@ -23,24 +25,42 @@ type WebHookConfigSpec struct {
 }
 
 type Webhook struct {
-	Type           string    `yaml:"type"`
-	SubscriptionID string    `yaml:"subscriptionID"`
-	SendFinished   bool      `yaml:"sendFinished"`
-	SendStarted    *bool     `yaml:"sendStarted,omitempty"`
-	EnvFrom        []EnvFrom `yaml:"envFrom"`
-	Requests       []string  `yaml:"requests"`
+	Type           string        `yaml:"type"`
+	SubscriptionID string        `yaml:"subscriptionID"`
+	SendFinished   bool          `yaml:"sendFinished"`
+	SendStarted    *bool         `yaml:"sendStarted,omitempty"`
+	EnvFrom        []EnvFrom     `yaml:"envFrom"`
+	Requests       []interface{} `yaml:"requests"`
```

Comment:
why interface{}

Reply:
we currently need to support both versions of webhook-configuration (`alpha1` and `beta1`). In `alpha1` the parameter is a `[]string`, in `beta1` the parameter is a `[]Request`. If we do not want to create 2 separate structures for this, it needs to be done this way (at least I don't know about any cleaner way). Do you have some suggestions how to make it better ?

Reply:
I'd suggest to introduce different packages (alpha1, beta1) each just containing the exact model that is used for that version. Like it's done in e.g. kubernetes source: https://github.com/kubernetes/api/tree/master/apps

Reply:
After a discussion with @warber we came to a conclusion that there is no simple way to implement this feature with a different packages. It will make more problems than advantages. From this reason we are sticking for the current implementation for now.